clang/unittests/Format/FormatTest.cpp

```diff
@@ -7987,8 +7991,8 @@ TEST_F(FormatTest, BreakConstructorInitializersAfterColon) {
       "    aaaaaaaaaaaaaaa(aaaaaaaaaaaa) {}",
       Style);
   verifyFormat("Constructor(aaaaaa aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa,\n"
-               "            aaaaaa aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa) :\n"
```

Comment:
I know we usually don't change existing tests. But I sincerely think this a bug. From the documentation I don't see any reason why there should be a line break.